Helper/Webhook.php

```diff
@@ -183,9 +183,9 @@ class Webhook
      */
     private $paymentMethodsHelper;
     /**
-     * @var InvoiceResourceModel
+     * @var InvoiceRepositoryInterface
      */
-    private $invoiceResourceModel;
+    private $InvoiceRepositoryInterface;
```

Comment:
Hi @FrankStappers,

Thanks a lot for opening this PR.

One small note. Could you change the name of the private data member to camel case?